package.json

```diff
@@ -203,6 +203,13 @@
           "type": "string",
           "default": "",
           "order": 1
+        },
+        "compileOnSave": {
+          "title": "Compile on Save",
+          "description": "Compile the current package when a file is saved and provide linter errors.",
```

Comment:
You should clarify that if this is disabled `runTestsOnSave` will also have no effect. In general, I don't like conflating the two and think it would be more understandable for people to select each option independently.